ResourcesCard.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import { CalloutCard } from "component/cards/CalloutCard";
+
+import libraryIcon from "./library.png";
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // TODO
+  useEffect(async () => {
+    let req = await getResources(userId);
+    setResources(req.resources);
+
+    let primaryResource = resources[0];
```

Comment:
this might be able to be simplified to [primaryResource, ...secondaryResources] assuming that there will only every be 6 elements in the resources array.